src/Qowaiv.Validation.DataAnnotations/Attributes/MultipleOfAttribute.cs

```diff
@@ -21,52 +21,39 @@ private MultipleOfAttribute(decimal factor)
     /// <summary>The factor of which the value has to be a multiple of.</summary>
     public decimal Factor { get; }
 
-    /// <inheritdoc />
-    public override bool RequiresValidationContext => true;
-
-    /// <inheritdoc />
-    [Pure]
-    protected override ValidationResult IsValid(object? value, ValidationContext validationContext)
-        => IsValid(value)
-        ? ValidationResult.Success!
-        : ValidationMessage.Error(FormatErrorMessage(validationContext.DisplayName), validationContext.MemberNames());
-
     /// <inheritdoc />
     [Pure]
     public override string FormatErrorMessage(string? name)
         => string.Format(CultureInfo.CurrentCulture, ErrorMessageString, name, Factor);
 
     /// <inheritdoc />
     [Pure]
-    public override bool IsValid(object? value) => value switch
-    {
-        // should be handled by required.
-        null => true,
-        string str => string.IsNullOrEmpty(str),
-
-        float flt => IsValid(flt),
-        double dbl => IsValid(dbl),
-        decimal dec => IsValid(dec),
-        Amount amount => IsValid((decimal)amount),
-        Money money => IsValid((decimal)money.Amount),
-        Percentage percentage => IsValid(100 * (decimal)percentage),
-        IConvertible convertible => IsValid(convertible),
-
-        _ => false,
-    };
+    public override bool IsValid(object? value)
+        => value is null
+        || value switch
```

Comment:
Why did you remove string?

Reply:
After some consideration: a string is an invalid type, it should not be successful.